### PR TITLE
blog post introducing Victor and Laura

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

**Review by yabellini — COMMENTED** [review]
Adding data that champions and mentors send to me.

**Comment by yabellini** [comment]
Hi @ldecicco-usgs and @BroVic, this is the blog post introducing you both as team on the champions program. I copy here the info you send me using Airtable form.

Laura, perhaps you want to add some more info about you? 

And you can also add some info about the project you will be working on after the text that introduce you both.  

Please, do any changes you want to.    The potential publication date is April 20.

**Comment by ldecicco-USGS** [comment]
Hi @yabellini , I think the first notification I got about this went to my junk folder. I think it looks great as-is.

File: content/blog/2023-05-11-champions-program-teams/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+</br>
+
+## Laura DeCicco - Mentor
+
+{{< figure src = "/img/team/lukas-mentor.jpeg" class = "pull-left" width = "300" alt = "Profile photo of Laura DeCicco" caption = "<center><strong>Laura DeCicco</strong></center>">}}
+
+Laura comes from Madison, WI USA. She works for the United States Geological Survey developing and maintaining several R packages. She is excited to be a mentor for the inaugural cohort of rOpenSci Champions.
+
+_We're excited to have Victor's project on the Champions program and Laura as his mentor.  We can't wait to see the results of the work of this team. Stay-tuned for the next team introduction_ .
```
Comment (steffilazerte) [inline content/blog/2023-05-11-champions-program-teams/index.md:39]:
```suggestion
_We're excited to have Victor in the Champions program and Laura as his mentor.  We can't wait to see the results of the work of this team. Stay-tuned for the next team introduction_.
```

File: content/blog/2023-05-11-champions-program-teams/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+*In this series of blog posts, we introduce you to the ten teams of this first cohort and what they will be working on in the program.*
+
+Meet **Victor Ordu** from Nigeria and **Laura DeCicco** from United States!
+
+
+## Victor Ordu - Champion
+
+{{< figure src = "/img/team/victor-ordu-mentee.jpg" class = "pull-left" width = "300" alt = "Profile photo of Victor Ordu" caption = "<center><strong>Victor Ordu</strong></center>">}}
+
+Victor Ordu is a medical doctor with many years of experience in the Nigerian healthcare space. Having spent the first ten years of his career in clinical medicine he found a niche in health policy and environmental health regulation when he switched to the public health domain.
```
Comment (steffilazerte) [inline content/blog/2023-05-11-champions-program-teams/index.md:25]: Is it okay that both bios are in third person?

File: content/blog/2023-05-11-champions-program-teams/index.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Victor has worked in various capacities in both the public and private sectors including as a field consultant for the Global Polio Eradication Initiative in the WHO EMRO Region. Public health exposed him to the centrality of data management in determining health outcomes.  The quest for the right tooling for the job began with EpiInfo and SPSS until he discovered R and instantly fell in love with the language. Since then he has not looked back. 
+
+Now an independent consultant for the UN and international NGOs working in Nigeria, R has been the focus of much of his work. He also conducts R training programs for public and private organizations in Nigeria. Victor is an alumnus of the University of Nigeria and has also acquired certifications from the University of Heidelberg (Germany), the World Bank Institute, and Galilee Management Institute (Israel), to mention a few. He is an active member of the R community and has authored R packages that have found great use among R enthusiasts worldwide.
```
Comment (steffilazerte) [inline content/blog/2023-05-11-champions-program-teams/index.md:29]: We could link to one or two of Victor's packages?

File: content/blog/2023-05-11-champions-program-teams/index.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+</br>
+
+## Laura DeCicco - Mentor
+
+{{< figure src = "/img/team/lukas-mentor.jpeg" class = "pull-left" width = "300" alt = "Profile photo of Laura DeCicco" caption = "<center><strong>Laura DeCicco</strong></center>">}}
```
Comment (steffilazerte) [inline content/blog/2023-05-11-champions-program-teams/index.md:35]: Update photo link

Comment (steffilazerte) [inline content/blog/2023-05-11-champions-program-teams/index.md:35]: Update photo, I think this is correct?
```suggestion
{{< figure src = "/img/team/laura-decicco.jpg" class = "pull-left" width = "300" alt = "Profile photo of Laura DeCicco" caption = "<center><strong>Laura DeCicco</strong></center>">}}
```

File: content/blog/2023-05-11-champions-program-teams/index.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+---
+title: "rOpenSci Champions Program Teams: Meet Victor Ordu and Laura DeCicco"
+author:
+  - Victor Ordu
```
Comment (steffilazerte) [inline content/blog/2023-05-11-champions-program-teams/index.md:4]: Victor needs an author file

File: content/blog/2023-05-11-champions-program-teams/index.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  - Laura DeCicco
+date: '2023-05-11'
```
Comment (steffilazerte) [inline content/blog/2023-05-11-champions-program-teams/index.md:6]: Reminder to update this and the folder date as needed :)

**Review by steffilazerte — COMMENTED** [review]
Good to go! Just a couple of tweaks for Laura's photo and Victor's author file.

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok **OR sooner?**
* [X] alternative text of images is informative
* [ ] author metadata is provided with correct folder name -> **Need one for Victor**
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

File: content/blog/2023-05-11-champions-program-teams/index.md (hunk 3 above)
Comment (yabellini) [inline content/blog/2023-05-11-champions-program-teams/index.md:29]:
```suggestion
Now an independent consultant for the UN and international NGOs working in Nigeria, R has been the focus of much of his work. He also conducts R training programs for public and private organizations in Nigeria. Victor is an alumnus of the University of Nigeria and has also acquired certifications from the University of Heidelberg (Germany), the World Bank Institute, and Galilee Management Institute (Israel), to mention a few. He is an active member of the R community and has authored R packages, like [naijR](https://github.com/BroVic/naijR) that have found great use among R enthusiasts worldwide.
```

File: content/blog/2023-05-11-champions-program-teams/index.md (hunk 6 above)
Comment (yabellini) [inline content/blog/2023-05-11-champions-program-teams/index.md:6]:
```suggestion
date: '2023-05-11'
slug: "intro-champion-program-team-victor-laura"
```

**Pull request merged**